CHANGELOG.md

```diff
@@ -5,6 +5,193 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.13.0 (2016-12-08)
+
+### Builder
++ Add capability to specify images used as a cache source on build. These images do not need to have local parent chain and can be pulled from other registries [#26839](https://github.com/docker/docker/pull/26839)
++ (experimental) Add option to squash image layers to the FROM image after successful builds [#22641](https://github.com/docker/docker/pull/22641)
+* Fix dockerfile parser with empty line after escape [#24725](https://github.com/docker/docker/pull/24725)
+- Add step # on `docker build` [#24978](https://github.com/docker/docker/pull/24978)
```

Comment:
Should this be `Add step number on …` (makes it a little more readable, at least for me 👼)